app/components/show/header_component.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="d-flex justify-content-between">
```

Comment:
Extracted show page header into component.

Reply:
@justinlittman Is the idea that this header component will be used for both work show and collection show?

Reply:
Yes, I plugged it into both. (I decided to extract after realizing I was copying from work show to collection show ...)